release-notes/topics/hub-442.adoc

```diff
@@ -0,0 +1,8 @@
+[[hub-442-intro]]
+= Automation Hub 4.4.2
```

Comment:
Version of Hub is 4.5.0, not 4.4.2 for AAP 2.2

Reply:
Fixed.